Exercises/fortran/01/Begin/CMakeLists.txt

```
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.16)
+project(KokkosFortranTutotial01 LANGUAGES Fortran C CXX)
+include(../../../common.cmake)
+
+find_package(flcl)
+
+add_executable(exercise exercise.f90 axpy-view-f.f90 axpy-view-cxx.cc)
+set_target_properties(
+    exercise
+    PROPERTIES
+        Fortran_MODULE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/mod"
+)
+target_include_directories(exercise
+    PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/mod>
+    INTERFACE
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/mod>
+        $<INSTALL_INTERFACE:mod>
+)
+target_include_directories(exercise
+    PRIVATE
+        ${CMAKE_CURRENT_SOURCE_DIR}
+)
+target_link_libraries(exercise ${PROJECT_LIBS} flcl::flcl)
```

```suggestion
target_link_libraries(exercise flcl::flcl)
```

I don't think this variable exists.